### PR TITLE
Stock-Market-Trends-Webapp

### PR DESCRIPTION
**Pull request by BertrandConxy**
# Stock-Market-Trends-Webapp

## Hello ✋ 

In this branch, I have built the whole website. Some of the points I highlight are:

- I initiated the repository with the create-react-app package
- I set up the redux state manager
- I fetched the data from Financial modeling API
- I built different components that make up the home page
- It is important to note that there is also the search input in order to search through companies by their name
- I set up routes so that when the user clicks on one of the company cards, It will take him/her to its details page
- Building the details page, I fetched the company profile data and its financial statements and render them altogether
- After all, I have done the tests, starting with pure functions unit tests in the redux actions file and then followed by snapshot tests for important components and later intergration.
- I deployed the app via Netlify [live demo](https://stockmarkettrends.netlify.app/)
- I made the presentation video [Loom video](https://www.loom.com/share/31ffe7df3df54f7ca8ab5d85c346a271)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from './pages/Home';
+import Details from './pages/Details';
+
```
Comment (rloterh) [inline src/App.js:4]: - [x] The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
image


**The original design:**
image

File: src/redux/stocks/stock.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/* eslint-disable max-len */
```
Comment (rloterh) [inline src/redux/stocks/stock.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Bertrand, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍 
- Descriptive Pull Request
- Correct gitflow implemented
- App is correctly deployed
- Good video presentation of the project
- Redux is effectively used to manage state data
- Good ReadMe
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thank you very much @rloterh. I am going to implement that very soon.

File: src/App.js (hunk 1 above)
Comment (BertrandConxy) [reply src/App.js:4]: I believe that it is implemented as you suggested in the recent commit I made 😃 👍

File: src/redux/stocks/stock.js (hunk 2 above)
Comment (BertrandConxy) [reply src/redux/stocks/stock.js]: It has been corrected in the recent commit that I made 👍

**Review by DeliceLydia — APPROVED** [review]
## Hi @BertrandConxy, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations🎉


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by BertrandConxy** [comment]
Thank you very much @DeliceLydia. I really appreciate your review..

**Pull request merged**